test/tests/functional/pbs_job_default_group.py

```diff
@@ -53,13 +56,14 @@ def test_job_with_default_group(self):
         """
         if self.server.hostname == self.mom.hostname:
             self.skipTest("Server and Execution host must be different")
-        attr = {'flatuid': True}
-        self.server.manager(MGR_CMD_SET, SERVER, attr)
         # add a temporary new user on execution host
-        cmd = "useradd -m testpbsuser1"
+        self.user_name =  'temppbs' + ''.join(random.choices(string.ascii_letters, k=5))
+        cmd = f"useradd -m {self.user_name}"
```

Comment:
This still will not solve the problem of having a leftover account in the system when the test is prematurely killed like kill -9. Then there could be a lot of random accounts left behind in the system. Why not just stick with the one 'testpbsuser1' account, and check if it already exists then remove from a default group.  I think you can run 'usermod' to do this. Then worst case is you might just have only 1 leftover account. and the test doesn't error out if the original 'useradd -m' fails.

Reply:
A kill -9 of the test case is what I'm worried about, or if the system suddenly crashes/shuts down, and there'll be no chance to call TearDown().

Reply:
Interesting point.  I have never thought about someone doing a kill -9 to a test.  Usually doing a kill -TERM or kill -INT will be better.  In any case, I like your idea of using one account.  At the start of the test, check if it is in the appropriate state, and fix it if it isn't.  This is basically what happens with PTL.  In setUp(), we call revert_to_defaults().  This is kind of the same thing.

Reply:
Got it. will update the test.